### PR TITLE
[v6r18] Improving SiteDirector code quality

### PR DESCRIPTION
**Pull request by vbpnl**
New PR

### CODE REVIEW

**Comment by chaen** [comment]
```New PR``` will make Andre angry :D

File: WorkloadManagementSystem/Agent/SiteDirector.py [hunk 1]
```diff
@@ -48,6 +48,9 @@
 MAX_JOBS_IN_FILLMODE = 5
 
 def getSubmitPools( group = None, vo = None ):
+  """
+    This method gets submit pools
```
Comment (chaen) [inline WorkloadManagementSystem/Agent/SiteDirector.py]: Can you please add the correct doctstring parameters (with ```param``` and ```return```)

Comment (petricm) [inline WorkloadManagementSystem/Agent/SiteDirector.py]: Wait, just after @andresailer will teach us how to do it!

**Comment by andresailer** [comment]
There is a binary file added to the commit `.DS_Store`
That should be dropped from the commit that introduced it...

**Comment by fstagni** [comment]
And, since you are there already, please add .DS_Store in the `.gitignore` file.

File: WorkloadManagementSystem/Agent/SiteDirector.py (hunk 1 above)
Comment (atsareg) [inline WorkloadManagementSystem/Agent/SiteDirector.py]: Did @andresailer manage to teach us to make correct doc strings ?

Comment (andresailer) [inline WorkloadManagementSystem/Agent/SiteDirector.py]: No, didn't manage

File: DataManagementSystem/Client/FTSFile.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def tableDesc():
                "FileID": "INTEGER NOT NULL",
                "OperationID": "INTEGER NOT NULL",
                "RequestID": "INTEGER NOT NULL",
-               "LFN": "VARCHAR(255) NOT NULL",
+               "LFN": "VARCHAR(955) NOT NULL",
```
Comment (chaen) [inline DataManagementSystem/Client/FTSFile.py]: Any reason why it is not 1024 like the others ?

Comment (vbpnl) [inline DataManagementSystem/Client/FTSFile.py]: LFN by design is smaller length than any SURL.
Any LFN with 1024 char will not fit in SURL.
955 is not an ideal choice either. Suggestions are welcome.

Comment (chaen) [inline DataManagementSystem/Client/FTSFile.py]: Well, 1024 is anyway a very large number, and I would definitely stick to power of 2, so I would be in favor of having both at 1024. If someone feels the need for more (Which I would find surprising anyway), they can always increase in their DB

**Pull request merged**